include/fw_info.h

```diff
@@ -212,12 +226,25 @@ static inline const struct fw_info *fw_info_check(u32_t fw_info_addr)
 	return NULL;
 }
 
+static inline const struct fw_info_abi_getter_info
+	*abi_getter_info_check(struct fw_info_abi_getter_info *abi_getter)
+{
+	const u32_t abi_getter_magic[] = {ABI_GETTER_MAGIC};
+
+	if (memeq(abi_getter->magic, abi_getter_magic,
+					CONFIG_FW_INFO_MAGIC_LEN)) {
+		return abi_getter;
+	}
+	return NULL;
+}
 
 /* The supported offsets for the fw_info struct. */
-#define FW_INFO_OFFSET0 0x200
-#define FW_INFO_OFFSET1 0x400
-#define FW_INFO_OFFSET2 0x800
-#define FW_INFO_OFFSET_COUNT 3
+#define FW_INFO_OFFSET0 0x0
+#define FW_INFO_OFFSET1 0x200
+#define FW_INFO_OFFSET2 0x400
+#define FW_INFO_OFFSET3 0x800
+#define FW_INFO_OFFSET4 0x1000
+#define FW_INFO_OFFSET_COUNT 5
```

Comment:
why do we now have 5?

Reply:
It's one of the changes (Add more supported offsets).

Reply:
okei thanks for clearing it up :)